playbooks/deploy-rock.yml

```diff
@@ -1136,6 +1136,17 @@
       group: root
     when: with_kibana
 
+  - name: Set initial Kibana credentials
+    shell: >
+      kibuser=$(getent passwd 1000 | awk -F: '{print $1}')
```

Comment:
Any user created by initial setup will be 1000, unless someone has really monkeyed with the ISO creation.  But in our build, the first user created will always be 1000.